Tests/Integration/config/messenger.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+    messenger:
+        transports:
+            async: 'in-memory://'
+        routing:
+            'Presta\SitemapBundle\Messenger\DumpSitemapMessage': async
+
+services:
```

Comment:
Is this required ?
Maybe Im' missing something, but the service should have been registered by the bundle.